README.md

```diff
@@ -2,3 +2,39 @@ Instant Share
 =============
```

Comment:
Yes, I agree, that part is not yet addressed. Let's do it in a separate PR. This PR attempts to fix the issue of describing why Instant Share is cool and useful, and what it does.